models/bow_model.ipynb

```diff
@@ -0,0 +1,564 @@
+     },
+     "execution_count": 104,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "# Logistic regression time!\n",
+    "logit = LogisticRegression(solver=\"lbfgs\", n_jobs=-1, random_state=7)\n",
+    "logit.fit(X_train, y_train)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 107,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "(0.877, 0.877)"
+     },
+     "execution_count": 107,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "round(logit.score(X_train, y_train), 3), round(logit.score(X_test, y_test), 3),"
```

Comment:
I got a ValueError here:
```
{
	"name": "ValueError",
	"message": "Found input variables with inconsistent numbers of samples: [520000, 2]",
	"stack": "---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/Users/Will/Documents/ECE 539 Final Project/ECE539-Final-Project/models/bow_model.ipynb Cell 15 line 1
----> <a href='vscode-notebook-cell:/Users/Will/Documents/ECE%20539%20Final%20Project/ECE539-Final-Project/models/bow_model.ipynb#X20sZmlsZQ%3D%3D?line=0'>1</a> round(logit.score(X_train, y_train), 3), round(logit.score(X_test, y_test), 3),

File ~/Documents/ECE 539 Final Project/ECE539-Final-Project/.venv/lib/python3.11/site-packages/sklearn/base.py:706, in ClassifierMixin.score(self, X, y, sample_weight)
    681 \"\"\"
    682 Return the mean accuracy on the given test data and labels.
    683 
   (...)
    702     Mean accuracy of ``self.predict(X)`` w.r.t. `y`.
    703 \"\"\"
    704 from .metrics import accuracy_score
--> 706 return accuracy_score(y, self.predict(X), sample_weight=sample_weight)

File ~/Documents/ECE 539 Final Project/ECE539-Final-Project/.venv/lib/python3.11/site-packages/sklearn/utils/_param_validation.py:214, in validate_params.<locals>.decorator.<locals>.wrapper(*args, **kwargs)
    208 try:
    209     with config_context(
    210         skip_parameter_validation=(
    211             prefer_skip_nested_validation or global_skip_validation
    212         )
    213     ):
--> 214         return func(*args, **kwargs)
    215 except InvalidParameterError as e:
    216     # When the function is just a wrapper around an estimator, we allow
    217     # the function to delegate validation to the estimator, but we replace
    218     # the name of the estimator by the name of the function in the error
    219     # message to avoid confusion.
    220     msg = re.sub(
    221         r\"parameter of \\w+ must be\",
    222         f\"parameter of {func.__qualname__} must be\",
    223         str(e),
    224     )

File ~/Documents/ECE 539 Final Project/ECE539-Final-Project/.venv/lib/python3.11/site-packages/sklearn/metrics/_classification.py:220, in accuracy_score(y_true, y_pred, normalize, sample_weight)
    154 \"\"\"Accuracy classification score.
    155 
    156 In multilabel classification, this function computes subset accuracy:
   (...)
    216 0.5
    217 \"\"\"
    219 # Compute accuracy for each possible representation
--> 220 y_type, y_true, y_pred = _check_targets(y_true, y_pred)
    221 check_consistent_length(y_true, y_pred, sample_weight)
    222 if y_type.startswith(\"multilabel\"):

File ~/Documents/ECE 539 Final Project/ECE539-Final-Project/.venv/lib/python3.11/site-packages/sklearn/metrics/_classification.py:84, in _check_targets(y_true, y_pred)
     57 def _check_targets(y_true, y_pred):
     58     \"\"\"Check that y_true and y_pred belong to the same classification task.
     59 
     60     This converts multiclass or binary types to a common shape, and raises a
   (...)
     82     y_pred : array or indicator matrix
     83     \"\"\"
---> 84     check_consistent_length(y_true, y_pred)
     85     type_true = type_of_target(y_true, input_name=\"y_true\")
     86     type_pred = type_of_target(y_pred, input_name=\"y_pred\")

File ~/Documents/ECE 539 Final Project/ECE539-Final-Project/.venv/lib/python3.11/site-packages/sklearn/utils/validation.py:407, in check_consistent_length(*arrays)
    405 uniques = np.unique(lengths)
    406 if len(uniques) > 1:
--> 407     raise ValueError(
    408         \"Found input variables with inconsistent numbers of samples: %r\"
    409         % [int(l) for l in lengths]
    410     )

ValueError: Found input variables with inconsistent numbers of samples: [520000, 2]"
}
```

Reply:
I found the bug! You need to use the same CountVectorizer - it's fixed now